tests/app/platform_stats/test_rest.py

```diff
@@ -122,7 +122,7 @@ def test_validate_date_is_within_a_financial_year_when_input_is_not_a_date(start
 
 
 def test_get_usage_for_all_services(notify_db_session, admin_request):
-    org, org_2, service, service_2, service_3, service_sms_only, \
+    org, org_3, service, service_3, service_without_org, service_sms_only, \
```

Comment:
It took me a while to understand the change here - are we trying to align with the returned names ([here](https://github.com/alphagov/notifications-api/blob/6d9a505ba65679d81cde5fa9607e908a7cdd2985/tests/app/db.py#L1019))? It's surprising to have `org` and `org_3`, but no `org_2`. Could we:

- Add some explanation for the change in a commit message.
- Change the names so they exactly match the returned names.

Using indices to represent different things is a bit of a smell, but that seems a bit out-of-scope to improve in this PR. If we did want to make a small improvement, we could:

- Rename `service_x` with something more distinctive, like `service_with_email`.
- Stop returning orgs, and just fetch them from a service when they're needed.

Reply:
I did some refactoring to make this fixture and it's outputs more readable, and also to stop variable assignment being based on returns order.